### PR TITLE
Show game history on Trials Passage

### PR DESCRIPTION
**Pull request by kyleshay**
I don't know if this is possible with the API but if it is it would be a great addition.

### CODE REVIEW

**Comment by kyleshay** [comment]
could build something custom for it, but at least the data is here now...
screen shot 2016-10-03 at 2 47 29 pm
screen shot 2016-10-03 at 2 47 37 pm

**Comment by kyleshay** [comment]
we were already making the API call for moments of triumph (have you seen that call?, we get so much info)

something like https://github.com/DestinyItemManager/DIM/issues/145 would be a no-effort task.

screen shot 2016-10-03 at 2 51 00 pm

**Comment by kyleshay** [comment]
also, not exactly sure how to see if one was flawless... might be missing it in the API.

**Comment by delphiactual** [comment]
Type | Filled | Unfilled
------------------------------
Wins | cfd92b | 66653c
Losses | b12e21 | 371913

If you wanted to make it look like in game:
trials-complete-lighthouse

**Comment by kyleshay** [comment]
might have to sit on this till next weekend :/ trials data from the weekend is now coming back as reset. do want to have the flawless check in there for completion. 

can consider adding the circles. will look a lot better.

**Comment by kyleshay** [comment]
Went flawless but didn't have time to check the API to see if anything was different... sitting on this till next weekend -_-

Also, I went 12 and 0 (no Vigilant Disciple 😡 ), so DIM showed that I was `12 / 9`. If I add the circles I'll want to maybe show the 9 circles, then after that maybe `+ 3` or something. Maybe show more circles, but then it might be hard to count.

**Comment by delphiactual** [comment]
+1 `+x`

**Comment by kyleshay** [comment]
Ok I think I figured out what flag denotes flawless... but just to be sure, I saved the response so I can mock data out later if needed:
https://www.diffchecker.com/DNt7OSjr

rec

Since I have the mock data, should be easy to make those circles this week.

**Comment by kyleshay** [comment]
screen shot 2016-10-31 at 12 11 20 pm

screen shot 2016-10-31 at 12 13 46 pm

**Comment by delphiactual** [comment]
Hound told you before I could...

**Comment by bhollis** [comment]
That looks great!

File: app/scripts/move-popup/dimMoveItemProperties.directive.js [hunk 1]
```diff
@@ -100,6 +101,12 @@
   function MoveItemPropertiesCtrl($sce, $q, storeService, itemService, settings, ngDialog, $scope, $rootScope, dimFeatureFlags, dimDefinitions) {
     var vm = this;
 
+    vm.getNumber = function(objective) {
```
Comment (bhollis) [inline app/scripts/move-popup/dimMoveItemProperties.directive.js]: Remember, calling a function from within an Angular expression is super expensive. Could we precalculate this with a watch, or better, use a filter?

Comment (bhollis) [inline app/scripts/move-popup/dimMoveItemProperties.directive.js]: This is a good approach: http://stackoverflow.com/a/29552460/11284

**Comment by kyleshay** [comment]
Converted to use filter and support `+ N` additional wins

screen shot 2016-11-01 at 10 24 23 am

**Pull request merged**